sw/otbn/crypto/p256_ecdsa_sca.s

```diff
@@ -33,33 +33,66 @@ p256_ecdsa_verify:
   jal      x1, p256_verify
   ecall
 
-.bss
+.data
 
```

Comment:
@bilgiday @jadephilipoom Can you check the size of the binary? IIUC this change will put these constants in the OTBN binary which will increase the size of anything that it will be linked into.